### PR TITLE
merge deps upgrade and slog migration

### PR DESCRIPTION
**Pull request by istyf**

### CODE REVIEW

File: cmd/integration-cip-sdl/main.go [hunk 1]
```diff
@@ -15,78 +16,80 @@ import (
...
 	isEnabled := os.Getenv(featureKey) == "true"
 
+	logger := logging.GetFromContext(ctx)
+
 	if isEnabled {
-		logger.Info().Msgf("feature %s is enabled", feature)
+		logger.Info(fmt.Sprintf("feature %s is enabled", feature))
 	} else {
-		logger.Warn().Msgf("feature %s is not enabled", feature)
+		logger.Warn(fmt.Sprintf("feature %s is enabled", feature))
```
Comment (shestyf) [inline cmd/integration-cip-sdl/main.go:80]: should this say that it is enabled or not? surely not both branches should have the same message?

File: internal/pkg/application/facilities/sportsfields.go [hunk 2]
```diff
@@ -30,10 +29,10 @@ func (s *storageImpl) StoreSportsFieldsFromSource(ctx context.Context, ctxBroker
 
 	for _, feature := range featureCollection.Features {
 		if feature.Properties.Type == "Aktivitetsyta" {
-			sportsField, err := parseSportsField(logger, feature)
+			sportsField, err := parseSportsField(ctx, feature)
 			if err != nil {
 				if !errors.Is(err, ErrSportsFieldIsOfIgnoredType) {
-					logger.Error().Err(err).Msg("failed to parse aktivitetsyta")
+					logger.Error("failed to parse aktivitetsyta", "err", err.Error())
```
Comment (shestyf) [inline internal/pkg/application/facilities/sportsfields.go:35]: should aktivitetsyta maybe be translated to sportsfield?

File: cmd/integration-cip-sdl/main.go (hunk 1 above)
Comment (istyf) [reply cmd/integration-cip-sdl/main.go:80]: good spot, will fix

File: internal/pkg/application/facilities/sportsfields.go (hunk 2 above)
Comment (istyf) [reply internal/pkg/application/facilities/sportsfields.go:35]: The swedish feature name has been used previously, but I agree that it looks odd. New changes incoming.

**Pull request merged**